src/responses/search-response-details.ts

```diff
@@ -102,6 +130,10 @@ export class SearchResponseDetails {
     if (body?.collection_extra_info) {
       this.collectionExtraInfo = body.collection_extra_info;
     }
+
+    if (body?.account_extra_info) {
+      this.accountExtraInfo = body.account_extra_info;
```

Comment:
can we do:
```suggestion
      this.accountExtraInfo = body.account_extra_info ?? null;
```

Reply:
Sure, I suppose we can change the type of the property to `AccountExtraInfo | null`.